docs/content/preview/yugabyte-platform/administer-yugabyte-platform/high-availability.md

```diff
@@ -112,61 +114,107 @@ After the active instance has been configured, you can configure one or more sta
 
 Your standby instance is now configured.
 
+### Verify HA
+
+To confirm communication between the active and standby, you can do the following:
+
+- Click **Make Active** on the standby (but don't promote it). You should see a list of available backups that you can restore from.
```

Comment:
Can we add the relevant image in the internal doc to clarify what we are talking about